assets/scss/components/stencil/productView/_productView.scss

```diff
@@ -119,17 +119,9 @@
 
 // Details
 // -----------------------------------------------------------------------------
-//
-// 1. for Androind Chrome and IE with zoom horizontal scroll fix
-//
-// -----------------------------------------------------------------------------
 
 .productView-details {
     padding-bottom: spacing("single") + spacing("third");
-
-    &.product-options {
-        overflow: hidden; // 1
-    }
```

Comment:
Could you please verify that this issue is not returned?

Reply:
Confirming